python/kserve/pyproject.toml

```diff
@@ -42,7 +42,7 @@ kubernetes = ">=23.3.0"
 python-dateutil = "^2.8.0"
 numpy = "^1.23.5"
 psutil = "^5.9.0"
-ray = { version = "^2.8.0", extras = ["serve"] }
+ray = { version = ">=2.8.1", extras = ["serve"] }
```

Comment:
hi :wave: why was the upper cap removed?

poetry `^2.8.1` == pip `>=2.8.1,<3`

Reply:
also, to propagate a version bump in the base package to the dependent packages' lockfiles, `make poetry-update-lockfiles` was recently added (and would be needed here)

Reply:
> hi 👋 why was the upper cap removed?
> 
> poetry `^2.8.1` == pip `>=2.8.1,<3`

Fixed here #3389

Reply:
awesome!

Reply:
many thanks @sivanantha321